### DISCUSSION TITLE
我在上个月支付了一年的费用 现在依旧显示即将到期

### DISCUSSION THREAD

**Discussion by farhigh233** [getlantern/lantern-forum-cn#771] [Q&A]
我在上个月支付了一年的费用 现在依旧显示即将到期
![微信图片_20240701213543](https://github.com/user-attachments/assets/31c5259c-496f-44f8-89f7-c99ae8c4ad0e)

**Comment by farhigh233** [comment]
账号231879414@qq.com

**Comment by MaxMKMax** [comment]
@farhigh233 
您好. 请放心, 我帮您提交和解决问题..
